### PR TITLE
Improve Overall UI of the system

### PR DESCRIPTION
**Pull request by frozenhelium**
- Update GO UI icon
- Update card components to use container
- Add vertical variant to Navigation Tab
- Add a prop to indicate parent route in the Navigation Tab
- Update List component to have a wrapper div
- Improve UI for surge pages

### CODE REVIEW

File: src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -6,8 +6,7 @@ import DefaultCheckmark, { CheckmarkProps } from './Checkmark';
 import styles from './styles.module.css';
 
 export interface Props<N> {
-    className?: string;
-    checkmark?: (p: CheckmarkProps) => React.ReactElement;
+    className?: string; checkmark?: (p: CheckmarkProps) => React.ReactElement;
```
Comment (tnagorra) [inline src/components/Checkbox/index.tsx]: Let's undo this change.

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by samshara — APPROVED** [review]
